### PR TITLE
Restore writing OperationKind during generation

### PR DESCRIPTION
**Pull request by 333fred**
Thanks @alrz for pointing this out.

### CODE REVIEW

**Comment by 333fred** [comment]
@dotnet/roslyn-compiler for a quick review. This hasn't actually affected any code yet, but it would be good to get it in quickly so it can flow into feature branches before people hit it.

**Review by cston — APPROVED** [review]

File: src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs [hunk 1]
```diff
@@ -156,6 +156,21 @@ private void WriteFiles()
                 }
             }
 
+            using (_writer = new StreamWriter(File.Open(Path.Combine(_location, "OperationKind.Generated.cs"), FileMode.Create)))
```
Comment (cston) [inline src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs:159]: >)) [](http://example.com/codeflow?start=126&length=2)

Do we need to set `Encoding.UTF8`?

**Review by AlekseyTs — APPROVED** [review]
LGTM (commit 1) assuming that a local testing has been performed verifying the expected behavior.

File: src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs (hunk 1 above)
Comment (333fred) [reply src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs]: Yes, thanks.

**Comment by 333fred** [comment]
@jaredpar for approval to merge into 16.11.

**Review by jaredpar — APPROVED** [review]

**Pull request merged**